### DISCUSSION TITLE
"Feature request" individual ping pong synth notes

### DISCUSSION THREAD

**Discussion by anthonybzh** [SynthstromAudible/DelugeFirmware#1682] [Ideas]
Hi.my first post here. (I Hope this is the right place) I just installed the nightly firmware and the new fonctionality are impressive. Amazing work, it’s fantastic. Thank you all. There was a trick to force deluge to authorize individual ping pong synth notes (by successively changing from synth to kit and back again to synth view) this is unfortunately no longer possible. It would be great if this little trick/fonction came back(or a similar thing)  Thanks if you Can, and thanks if you can't ! Keep up the good work.

**Comment by seangoodvibes** [comment]
Added this into synth/midi/cv clips :)

Hold audition pad when entering the play direction menu and it will set the sequence direction for the note row selection

While you're in the menu you can also press other note rows to set the sequence direction for those note rows quickly

**Reply by anthonybzh** [reply]
Ok, thanks Sean, i will try it soon . 🙂
